robot-db/src/main/java/repositories/BookInput.java

```diff
@@ -0,0 +1,62 @@
+package repositories;
+
+import dao.BookDAO;
+import domain.Author;
+import domain.Book;
+import domain.Category;
+import domain.PrintHouse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Class responsible for Inserting Parsed Book into Database
```

Comment:
DoEs CaSe MaTTeR? iS iT imPorTanT? or not? Now I wonder. In legal documents, now "Inserting Parsed Book into Database" has special meaning. Here?